components/collapsible-panel/collapsible-panel.js

```diff
@@ -427,22 +424,32 @@ class CollapsiblePanel extends SkeletonMixin(FocusMixin(RtlMixin(LitElement))) {
 	}
 
 	_renderHeader() {
+		const expandCollapseLabel = this.expandCollapseLabel || this.panelTitle;
 		return html`
 			<div class="d2l-collapsible-panel-header" @click="${this._handleHeaderClick}">
 				<div class="d2l-collapsible-panel-before">
 					<slot name="before" @slotchange="${this._handleBeforeSlotChange}"></slot>
 				</div>
 				<div class="d2l-collapsible-panel-header-primary">
-					${this._renderPanelTitle()}
-					<div class="d2l-collapsible-panel-header-actions" @click="${this._handleActionsClick}">
-						<slot name="actions"></slot>
-					</div>
-					<div class="d2l-collapsible-panel-opener">
+					<button
+						class="d2l-collapsible-panel-opener"
+						aria-expanded="${this.expanded}"
+						tabindex="0"
```

Comment:
We shouldn't need `tabindex="0"` since `button` elements are focusable by default.